website/content/docs/configuration/client.mdx

```diff
@@ -355,6 +355,100 @@ see the [drivers documentation](/docs/drivers).
   files on the client host via the `file` function. By default templates can
   access files only within the [task working directory].
 
+- `max_stale` `(string: "5s")` - # This is the maximum interval to allow "stale"
+  data. By default, only the Consul leader will respond to queries. Requests
+  to a follower will forward to the leader. In large clusters with many requests,
+  this is not as scalable, so this option allows any follower to respond to a query,
+  so long as the last-replicated data is within these bounds. Higher values result
+  in less cluster load, but are more likely to have outdated data.
```

Comment:
Thanks for the call out @tgross. 

Now that I am looking at the implementation and the original ticket a little deeper, I think I need to clarify the requirement. 

@schmichael  - Do we actually want to enforce a `5s` default for this or allow users to simply not set this value, and thus opt-in to the default `consul-template` behavior. The current logic only sets a value if the user specifies one, and this seems correct to me.

Reply:
Hm, I think a `5s` default is preferable because I suspect no one will think to configure this before they hit performance issues with Consul at which point they have to restart *all* of their Nomad agents to relieve some strain.

However, I hate changing default values like this in Z releases (where the 3 in 1.2.3 is the Z release), do you think we should hold this off until 1.3.0?

Reply:
Nevermind! I misunderstood the *existing* default behavior: I assumed `max_stale` was *disabled* by default but it is [2s](https://github.com/hashicorp/consul-template/blob/d63e1925f4432ab5aadf452c02244392105b50ae/config/config.go#L539) (thanks to @DerekStrickland for the correction!).

Let's use consul-template's default then by default `max_stale` to an empty string.